### PR TITLE
New topics

### PR DESCRIPTION
**Pull request by adambinch**
New topic `/topological_navigation/move_action_status` reports the name, status (PENDING, ACTIVE, ABORTED etc) and goal of the current move action, as a json string.
New topic `/closest_node_distance` reports the distance (Float32) of the robot to the closest node.

Both action servers report their terminal state as aborted (rather than succeeded) if the move action fails.
Better prints and logs from nav script.
Better default move base actions list in toponav launch file.
Some tidying.

### CODE REVIEW

**Comment by adambinch** [comment]
retest this please

**Comment by Jailander** [comment]
@marc-hanheide do you know why the checks are failing?

File: topological_navigation/src/topological_navigation/edge_action_manager.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class EdgeActionManager(object):
     def __init__(self):
         
         self.client = None        
-        self.current_action = None
+        self.current_action = "none"
```
Comment (Jailander) [inline topological_navigation/src/topological_navigation/edge_action_manager.py:60]: Are you sure about this? `None` is quite different to "none"` 
Are we checking `self.current_action` anywhere?

Comment (adambinch) [reply topological_navigation/src/topological_navigation/edge_action_manager.py:60]: Doesnt make a difference. The preempt function checks if the client is not None and if the client is not None then the current action is also not "none". Have you tested this? Does it break?

Comment (adambinch) [reply topological_navigation/src/topological_navigation/edge_action_manager.py:60]: So self.action name is not checked explicitly, it is checked by checking that the client is not None

**Comment by adambinch** [comment]
retest this please

**Review by francescodelduchetto — APPROVED** [review]

**Pull request merged**